assets/js/register.coffee

```diff
@@ -7,24 +7,41 @@ class Register
 
   constructor: (view) ->
     @view = view
-    @type = TYPES.NONE
+    @type = Register.TYPES.NONE
     return @
 
   saveChars: (chars) ->
-    @type = TYPES.CHARS
+    @type = Register.TYPES.CHARS
     @chars = chars
 
   saveRows: (serialized_rows) ->
-    @type = TYPES.ROWS
+    @type = Register.TYPES.ROWS
     @serialized_rows = serialized_rows
+  
+  serialize: () ->
+    data = ''
+    if @type==Register.TYPES.CHARS
```

Comment:
My usual take on style stuff is that I'll keep it in mind for future commits but don't want it to block progress. If you set up a linting system and a git hook that calls it, I would use that.